falcon/ms_io/mgf_io.py

```diff
@@ -41,20 +43,27 @@ def _parse_spectrum(spectrum_dict: Dict) -> sus.MsmsSpectrum:
     MsmsSpectrum
         The parsed cluster.
     """
-    identifier = spectrum_dict['params']['title']
+    identifier = spectrum_dict["params"]["title"]
 
-    mz_array = spectrum_dict['m/z array']
-    intensity_array = spectrum_dict['intensity array']
-    retention_time = float(spectrum_dict['params'].get('rtinseconds', -1))
+    mz_array = spectrum_dict["m/z array"]
+    intensity_array = spectrum_dict["intensity array"]
+    retention_time = float(spectrum_dict["params"].get("rtinseconds", -1))
 
-    precursor_mz = float(spectrum_dict['params']['pepmass'][0])
-    if 'charge' in spectrum_dict['params']:
-        precursor_charge = int(spectrum_dict['params']['charge'][0])
+    precursor_mz = float(spectrum_dict["params"]["pepmass"][0])
+    if "charge" in spectrum_dict["params"]:
+        precursor_charge = int(spectrum_dict["params"]["charge"][0])
     else:
-        raise ValueError('Unknown precursor charge')
+        precursor_charge = -1
```

Comment:
**todo:** This is not a good default charge value, because negative ion mode will be `-1`.

Reply:
Done